cmd/create/accountroles/creators.go

```diff
@@ -303,3 +303,10 @@ func buildAttachRolePolicyCommand(accRoleName string, policyARN string) string {
 		AddParam(awscb.PolicyArn, policyARN).
 		Build()
 }
+
+func getAccountRoleName(prefix string, roleName string) string {
```

Comment:
Not sure if this will be sufficient, as we are iterating over this map `aws.HCPAccountRoles` in other commands as well: `upgrade account roles`, `create cluster`, and `delete account roles`.

I was thinking of another approach to changing the value here https://github.com/openshift/rosa/blob/master/pkg/aws/policies.go#L103.

Also started this thread https://redhat-internal.slack.com/archives/C03SZLX3A10/p1684916329073869 to check if we can keep the `-Role` suffix to simplify flow, and be consistent with the other flows.